src/test/java/com/lpvs/entity/report/LPVSReportBuilderTest.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2024, Samsung Electronics Co., Ltd. All rights reserved.
+ *
+ * Use of this source code is governed by a MIT license that can be
+ * found in the LICENSE file.
+ */
+package com.lpvs.entity.report;
+
+import com.lpvs.entity.LPVSFile;
+import com.lpvs.entity.LPVSLicense;
+import com.lpvs.service.LPVSLicenseService;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.autoconfigure.thymeleaf.ThymeleafAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.thymeleaf.TemplateEngine;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@ImportAutoConfiguration(ThymeleafAutoConfiguration.class)
+@ContextConfiguration(classes = {LPVSReportBuilder.class})
+@Slf4j
+public class LPVSReportBuilderTest {
+
+    @Autowired private TemplateEngine templateEngine;
+
+    LPVSFile file1, file2, file3, file4;
+    LPVSLicense lic1, lic2, lic3, lic4;
+    LPVSLicenseService.Conflict<String, String> conflict1, conflict2;
+    LPVSReportBuilder reportBuilder;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        lic1 =
```

Comment:
It would be nice if the variable name is better named around the functionality like licMIT instead of simple number.